packages/web-components/fast-foundation/src/toolbar/toolbar.template.ts

```diff
@@ -34,7 +34,7 @@ export function toolbarTemplate(
             })}
         >
             <slot name="label"></slot>
-            <div class="positioning-region" part="positioning-region">
+            <div class="toolbar" part="toolbar">
```

Comment:
Here and radiogroup I don't know that this is more clear - because the entire component, the host element  - is the toolbar.

Reply:
Longer comment on `dialog.register`, standardizing on "control"